_includes/live-demos/drive-standalone-browse-controls/example.js

```diff
@@ -0,0 +1,17 @@
+var button = document.querySelector('.tinydrive-standalone-demo-browse');
+
+button.addEventListener('click', function () {
+  var controls = tinydrive.browse({
+    token_provider: '/your-local/jwt-provider'
+  }, function() {
```

Comment:
There's some inconsistencies in the spacing here, as some use `function () {` and others use `function() {`. Can we make them consistent please? I'm not sure which we normally use of the top of my head, but they should at least be consistent in the same demo.

- [ ] Fix inconsistent formatting